test/snippet/core/detail/persist_view.cpp

```diff
@@ -9,11 +9,14 @@ int main()
     // explicitly create an l-value of our dna vector:
     auto vec = "ACGT"_dna4;
     auto v = vec | seqan3::views::to_char;
```

Comment:
```suggestion
    [[maybe_unused]] auto v = vec | seqan3::views::to_char;
```
?

might be clearer?

Reply:
In all other examples we print them and therefore "use" them.

This was the only snippet where we use this technique.

Reply:
Yes, `[[maybe_unused]]` is better, I was a bit dispassionate here, because it's just detail :)